tests/docker/gitlab.Dockerfile

```diff
@@ -2,14 +2,18 @@
 FROM python:3.9
 
 ARG GCLOUD_VERSION=364.0.0
+ARG HELM_VERSION=3.5.4
 
 RUN apt-get update && apt-get install -y curl git jq \
 	&& curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
 	&& chmod +x ./kubectl \
 	&& mv ./kubectl /usr/local/bin \
 	&& curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GCLOUD_VERSION}-linux-x86_64.tar.gz \
 	&& tar xvzf google-cloud-sdk-${GCLOUD_VERSION}-linux-x86_64.tar.gz \
-	&& mv google-cloud-sdk /usr/lib/
+	&& mv google-cloud-sdk /usr/lib/ \
+	&& curl -LO https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz \
```

Comment:
I couldn't find changes related to this. Why do we need to install helm?

Reply:
The helm tests run on GKE because of networking issues in gitlab. Now that we have moved the GKE setup to the `before_script`, the helm tests run on the same runner as the GKE setup runner, and I thought it made more sense to have everything use the same image rather than have an entire separate logic for just helm.